public/index.html

```diff
@@ -333,6 +334,7 @@ <h1 class="logo"><a href="index.html#home">Onassis</a></h1>
     <!-- Bootstrap core JavaScript
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
+	<script src="events.js"></script>
```

Comment:
@wingchunkf FYI - a better naming convention for this file would be "ga.js" or "google_analytics.js" since in a project we could also have other _events_ such as login, logout, contact us, etc. and a file with such a broad name such as "events.js" could be confusing.